wire-ios-sync-engine/Source/Synchronization/EventProcessor.swift

```diff
@@ -66,13 +62,23 @@ actor EventProcessor: UpdateEventProcessor {
 
     func processEvents(_ events: [ZMUpdateEvent]) async throws {
         try await enqueueTask {
+
+            await MainActor.run {
```

Comment:
is it necessary? what about posting from current queue and choosing on receiving side to dispatch on mainQueue or other queue.

Reply:
I agree, fixed.